### DISCUSSION TITLE
i3 in Manjaro KDE gets stuck at the lockscreen

### DISCUSSION THREAD

**Discussion by codeleux** [i3/i3#5540] [Q&A]
The screen gets stuck at the kde's lockscreen when logging in, the commands doesn't work at all and i don't really know what's happening here since i'm new to the i3 thing.

##### Mouse works btw, i can move it through the screen.
![WhatsApp Image 2023-06-24 at 7 15 36 PM](https://github.com/i3/i3/assets/103602920/ca5d5684-1619-4312-8063-c139548e352d)

**Comment by arvyy** [comment]
It's not stuck, pressing super+enter for example should open a terminal. You see login screen because that is what was last rendered and i3 hasn't had a need to draw anything in that region yet. To have a proper desktop background in an empty workspace, look into packages like `feh` or `nitrogen` and invoke them from your i3 config with `exec`.
